lib/interactor.rb

```diff
@@ -163,4 +163,30 @@ def call
   # Returns nothing.
   def rollback
   end
+
+  private
+
+  # Internal: Determine what arguments (if any) should be passed to the "call"
+  # instance method when invoking an Interactor. The "call" instance method may
+  # accept any combination of positional and keyword arguments. This method
+  # will extract values from the context in order to populate those arguments
+  # based on their names.
+  #
+  # Returns an Array of arguments to be applied as an argument list.
+  def arguments_for_call # rubocop:disable Metrics/MethodLength
```

Comment:
Personally, I'd rather raise an exception for positional arguments. It would help avoiding such situations (just checked this in `call-with-keywords` branch):

```ruby
class A
  include Interactor
  def call(foo, bar: "")
    puts "Foo is #{foo}"
    puts "Bar is #{bar}"
  end
end

A.call(bar: 1)

# Foo is {:bar=>1}
# Bar is 
```

Also this would draw a clear line between approaches we support and ones we don't. 
@laserlemon, I'm open to discussion.